api/USB.json

```
@@ -40,6 +40,55 @@
           "deprecated": false
         }
       },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "tags": [
+            "web-features:webusb"
+          ],
+          "support": {
+            "chrome": [
+              {
+                "version_added": "118",
+                "partial_implementation": true,
+                "notes": "Available in dedicated workers and WebExtension service workers, not available in shared workers and normal service workers."
+              },
+              {
+                "version_added": "70",
+                "partial_implementation": true,
+                "notes": "Available in dedicated workers, not available in shared workers and service workers."
+              }
```

I guess the second note only applies for Chrome < 118?
```suggestion
              {
                "version_added": "118",
                "partial_implementation": true,
                "notes": "Available in dedicated workers and WebExtension service workers, not available in shared workers and normal service workers."
              },
              {
                "version_added": "70",
                "version_removed": "118",
                "partial_implementation": true,
                "notes": "Available in dedicated workers, not available in shared workers and service workers."
              }
```